android/app/webviewAssets.gradle

```diff
@@ -7,6 +10,45 @@
  * more information.
  */
 
+/**
+ * Windows support functions.
+ *
+ * On Windows, we have three different possible setups:
+ *   1. Windows Gradle + MSys Git's bash.
+ *   2. Windows Gradle + WSL's bash.
+ *   3. WSL Gradle + WSL's bash.
+ *
+ * Case 3, fortunately, should behave exactly like Linux.
+ *
+ * In cases 1 and 2, however, all paths Gradle provides are backslash-separated.
+ * This is problematic on its own, as the bash script we're invoking doesn't
+ * grok backslashes, so we normalize those on Windows. That's enough for case 1.
+ *
+ * Case 2 is worse, though. MSys's `bash` groks both "C:/" and "C:\"; more
+ * generally, it operates on the Windows filesystem with Windows path names. The
+ * WSL `bash` doesn't -- it's a standard Linux `bash` (or close enough), and it
+ * only sees the Windows disks as ordinary mounts under the lone volume '/'.
```

Comment:
I had to read this line a couple of times to understand what's meant by "the lone volume '/'". I think in the world where '/' is a thing, it doesn't make a lot of sense to call it a "volume" -- if anything that'd refer to one of the several things that might get mounted as filesystems at various paths in the overall path hierarchy that begins at '/'.

Would "ordinary mounts under the Unix-style single root path '/'" express what you mean here?

Or "only sees the Windows disks as ordinary mounts at paths like '/mnt/c/'" might add some helpful further concreteness.